code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -615,9 +615,15 @@
 		return TRUE
 	return FALSE
 
-/mob/living/carbon/human/can_sting()
-	if(species?.species_flags & IS_SYNTHETIC)
+/mob/living/can_sting()
+	if(isxeno(src)) //Xenos don't process reagents
 		return FALSE
+
+	if(ishuman(src))
+		var/mob/living/carbon/human/target = src
+		if(target.species?.species_flags & IS_SYNTHETIC)
+			return FALSE
```

Comment:
why is this being changed this was fine as is and this adds snowflake code

Reply:
Future proofing but I don't care about this enough to fight over it